earth_enterprise/src/fusion/portableglobe/servers/local/preview/developers/terms.html

```diff
@@ -1,6 +1,6 @@
 <!--
   Copyright 2017 Google Inc.
-  Copyright 2018 the OpenGEE Contributors.
+  Copyright 2018-2019 the OpenGEE Contributors.
```

Comment:
Does the addressed bug existed in previous version ? (5.2.5). If yes, let's document this fix in release notes.

Reply:
I did a checkout of OpenGEE branch release_5.2.5 and ran `getversion.py` on both Ubuntu and Windows.  Both reported 5.2.5.  So it appears the issue is either new to or first appeared with 5.3.0.  So no need for a release note, correct?

Reply:
Ok, then it's introduced in 5.3.0 and fixed in 5.3.0 and we don't need to document it.